pipeline/templates/mof.yaml

```diff
@@ -0,0 +1,17 @@
+parameters:
+  - name: azureDevOpsPAT
+  - name: numberOfAgents
+  - name: azureDevOpsURL
+
```

Comment:
can we change the name of this file to something more descriptive like

**compile-dsc.yaml**